src/Workspaces/CSharp/Portable/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -116,6 +116,15 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
                 }
             }
 
+            // * { - in the local function context
+            if (currentToken.IsKind(SyntaxKind.OpenBraceToken) && currentTokenParentParent != null && currentTokenParentParent.IsKind(SyntaxKind.LocalFunctionStatement))
```

Comment:
Nit: Since I see the rest of this code uses the same pattern

1. `IsKind` checks for a `null` receiver (sneaky extension method).
2. Even if it didn't, `?.` is a thing now :)